Interview/questions.md

```diff
@@ -105,19 +93,15 @@ Please keep in mind that this document is in a continuous state of flux, with ne
 - How familiar are you with reading crash reports?
 - Are you familiar with `Crashlytics`?
 - What analytics frameworks are you familiar with or have used?
-- Have you used `TestFlight`? `Fabric`?
+- Have you used `TestFlight`? `Firebase`?
 
 ### COCOA AND FRAMEWORKS
 
 - What is a `thread`? In which cases would you create your own threads?
-- What is a `runloop`? What is a `runloop mode`?
 - What do you think of `Grand Central Dispatch`?
 - Have you used `dispatch_once`? In broad strokes, how does it work?
-- Have you used `dispatch_barrier`? When is it appropriate and useful to use?
-- What is a `dispatch_group`?
-- Have you used `NSOperationQueue` and `NSOperation`?
+- What is a `dispatch group`?
 - What methods for thread synchronisation do you know on iOS?
-- What are the main differences between `GCD` queues and `NSOperationQueue`s?
 - What are the advantages and disadvantages of `GCD` and `NSOperationQueue`/`NSOperations`?
 - What is `KVO`? Do you know how it works?
```

Comment:
Do we need this?

Reply:
IMO it's more important to know about KVO than NSOperations...